torch/onnx/utils.py

```diff
@@ -1004,35 +1019,98 @@ def _run_symbolic_function(g, block, n, inputs, env, operator_export_type=Operat
             elif op_name == "device" and n.output().type().kind() == "DeviceObjType":
                 return None
             elif op_name == 'Loop' or op_name == 'If':
-                new_op_outputs = g.op(op_name, *inputs, outputs=n.outputsSize())
-                new_node = new_op_outputs[0].node() if n.outputsSize() > 1 else new_op_outputs.node()
-                for b in n.blocks():
-                    new_block = new_node.addBlock()
-                    # Copy input metadata to subblock
+                static_if = False
+                enable_const_folding = False
```

Comment:
Now we don't need `enable_const_folding`.
Previously we need it - we cannot do const folding in if sub-block (nested if). - because this PR has a bug, that is:
When we create const node, we append it using `new_block->owningGraph()->appendNode(const_node);`
Then it will append the const node into the graph. This is not correct for sub-block case.
It triggers the test failure for `pytest test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset11::test_floating_point_infer_dtype`
We should append it into the this node's corresponding block, e.g., `new_block->appendNode(const_node);`
After this fix, it is good. So we don't need  `enable_const_folding`.